tests-beta/ios/MASVS-STORAGE/MASTG-TEST-0x52-1.md

```diff
@@ -0,0 +1,31 @@
+---
+platform: ios
+title: Data Stored in Private Storage - Content Comparison Test
+id: MASTG-TEST-0x52-1
+type: [dynamic]
+weakness: MASWE-0006
```

Comment:
Please add `best-practices:` for all these tests, they will be similar if not the same I guess

Reply:
done